app/Models/User.php

```diff
@@ -1159,6 +1160,28 @@ static function (Builder $query): void {
             ->first();
     }
 
+    public function getPrimaryTeamAttribute(): ?Team
+    {
+        $teams = Attendance::where('gtid', $this->gtid)
+            ->where('attendable_type', Team::getMorphClassStatic())
+            ->leftJoin('teams', static function (JoinClause $join): void {
+                $join->on('teams.id', '=', 'attendance.attendable_id')
+                    ->where('attendable', '=', true);
+            })
+            ->where('attendable', '=', true)
+            ->groupBy('attendable_id')
+            ->select(
+                'attendable_id',
+                DB::raw('count(*) as count'),
+                DB::raw('\''.Team::getMorphClassStatic().'\' as attendable_type')
```

Comment:
Feels super weird to have code directly in raw SQL but I don't have a better suggestion.

Reply:
This is just to grab a string value